helm/eviction-autoscaler/templates/configmap.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "eviction-autoscaler.fullname" . }}-config
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/name: {{ include "eviction-autoscaler.name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/component: controller
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+data:
+  config.yaml: |-
+    metrics:
+      enabled: {{ .Values.controllerConfig.metrics.enabled }}
+
+    namespaces:
+      watchMode: {{ .Values.controllerConfig.namespaces.watchMode | quote }}
+{{- if and .Values.controllerConfig.namespaces.excludeList (gt (len .Values.controllerConfig.namespaces.excludeList) 0) }}
+      excludeList:
+{{ toYaml .Values.controllerConfig.namespaces.excludeList | indent 8 }}
+{{- end }}
+{{- if and .Values.controllerConfig.namespaces.includeList (gt (len .Values.controllerConfig.namespaces.includeList) 0) }}
+      includeList:
```

Comment:
I _think_ we may want to do these as annotations on the namepaces themselves.  But why do I think that

Maybe so that webhook can sutomatically turn this on? But why not use excludes.